pywbem/_recorder.py

```diff
@@ -279,15 +285,18 @@ def open_file(filename, file_mode='w'):
         done differently in pythong 2 and python 3
 
         Parameters:
-          filename: (:term: `string`):
```

Comment:
I wonder whether we even an `open_file()` method. I found the following code on http://python-future.org/compatible_idioms.html#file-io-with-open that is supposedly compatible between Python 2 and Python 3:
```
from io import open
f = open('myfile.txt', encoding='utf-8')
text = f.read()    # unicode, not bytes
```
Instead of having our own `open_file()` static method (which is not placed well as a static method on `BaseOperationRecorder` anyweay, given the generality of the method), we could show the compatible usage in the examples, and use it ourselves in the testsuite.

Reply:
I am fixing to keep the function but note that the alternative is to use open from io.